pages/nats/reference-content/limitations.mdx

```diff
@@ -0,0 +1,22 @@
+---
+meta:
+  title: NATS Limitations
+  description: Learn about the current limitations of Scaleway NATS
+content:
+  h1: NATS Limitations
+  paragraph: Learn about the current limitations of Scaleway NATS
+tags: messaging limitations space size storage payload max-streams max-consumers max-subscribers max-topics visibility-timeout
+dates:
+  validation: 2025-04-01
+  posted: 2025-04-01
+categories:
+  - serverless
+---
+
+## Usage limits
+
+For the usage limits that apply when using Scaleway NATS, see the dedicated sections on our [Organization quotas](/organizations-and-projects/additional-content/organization-quotas/) page. Limits (quotas) apply to, for example, the maximum number of queues, streams, topics and consumers/subscribers, as well as to message retention duration and storage.
```

Comment:
add anchor to NATS quotas maybe

Reply:
I will but can't yet, separating the quotas on the organisation quotas page will be something I do on the feature branch once the others are merged